modules/core/src/test/java/org/apache/ignite/internal/GridMBeansTest.java

```diff
@@ -60,6 +100,22 @@ public void testKernalBeans() throws Exception {
         checkBean("Kernal", "ClusterMetricsMXBeanImpl", "TotalServerNodes", 1);
     }
 
+    /** Check that eviction bean is available */
+    public void testEvictionPolicyBeans() throws Exception{
```

Comment:
Let's move this test case to a separate test (say, a new test called org.apache.ignite.internal.processors.cache.GridEvictionPolicyMBeansTest).